index.js

```diff
@@ -150,7 +150,7 @@ function parse(input, options) {
 	}
 
 	for (const param of input.split('&')) {
-		let [key, value] = param.replace(/\+/g, ' ').split('=');
+		let [key, value] = param.replace(/\+/g, ' ').split(/\=(.+)/); // eslint-disable-line no-useless-escape
```

Comment:
ESLint is correct. You don't have to escape `=`.

Reply:
@sindresorhus - yup I know, but then I get a different ESLint error about escaping.   Yup I will add a test.  Also, it seems to fail other tests now, so I have to look into it

Reply:
Ah ok. Better to ignore the `no-div-regex` rule.